R/run_geocoder.R

```diff
@@ -0,0 +1,257 @@
+#' Geocoding voter file addresses with coordinates (latitude and longitude) and/or census geographies.
+#'
+#'
+#' @param voter_file: A data frame contain the voter addresses, separated into columns for street, city, state, and zipcode
```

Comment:
These docstrings should make sure they don't pass 80 characters (can be wrapped to additional lines).